### PR TITLE
Clarified lack of Zstd compression codec in README.

### PR DESCRIPTION
**Pull request by vvvvalvalval**

### CODE REVIEW

**Comment by vvvvalvalval** [comment]
Related to issue: https://github.com/airlift/aircompressor/issues/109

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ typically 10-40% faster than the JNI wrapper for the native libraries.
 # Hadoop CompressionCodec
 
 In addition to the raw block encoders, there are implementations of the
-Hadoop CompressionCodec for each algorithm. They are
+Hadoop CompressionCodec for each algorithm (except Zstandard, see [issue #109](https://github.com/airlift/aircompressor/issues/109)). They are
```
Comment (martint) [inline README.md:15]: Let's list the algorithms for which the codec is implemented, instead.

**Comment by bondolo** [comment]
This appears to be resolved in #152

**Comment by dain** [comment]
Zstd Hadoop codec was implemented in #152

**Pull request closed without merging**